### ISSUE TITLE
Dropping support for Ruby 2.4 and 2.6

### ISSUE THREAD

**Issue by zmariscal** [zdennis/activerecord-import#828]
I recently opened a PR and noticed that this project is running tests for Ruby version 2.4 and 2.6. Is there a plan to support these indefinitely? According to [this](https://endoflife.date/ruby) it seems these have been EOL. For 2.4 it hasn't gotten an update since 2020, and 2022 for 2.6

**Comment by jkowens** [comment]
Yeah we should officially trim down support to later versions of Ruby and ActiveRecord.

**Comment by zmariscal** [comment]
Happy to make a PR for what I deem as reasonable. But if you want to remove certain parts feel free to tell me too 😄
